conflation/mapillary.py

```diff
@@ -22,14 +21,14 @@
 IMAGES_URL = "https://a.mapillary.com/v3/images?client_id={}&sequence_keys={}&per_page={}"
 
 
-def run(bbox: str, output_dir: str, output_tmp_dir: str, config: dict, processes: int) -> None:
+def run(bbox: str, traces_dir: str, tmp_dir: str, config: dict, processes: int) -> None:
```

Comment:
Any changes here is basically renaming / cleanup of folders and files on the disk.